### ISSUE TITLE
About the best setting of hyperparameters

### ISSUE THREAD

**Issue by ZQSong1997** [YangLiangwei/ConsisRec#2]
Hello, what is the best parameter setting（ batch size, learning rate and embedded dimension） for ciao and epinions data sets?
Thanks!

**Comment by sigaofei** [comment]
你运行出来了吗？运行就一直不动

**Comment by ZQSong1997** [comment]
> 你运行出来了吗？运行就一直不动

可以运行，作者的代码没问题，就是模型训练的时间比较长

**Comment by sigaofei** [comment]
能不能给个联系啊方式交流一下

**Comment by YangLiangwei** [comment]
之前的log文件已经找不到了，可以按照parameter sensitivity部分调一下参数，如果碰到什么问题 可以加我微信 18215561894
